sympy/functions/special/delta_functions.py

```diff
@@ -409,25 +409,23 @@ class Heaviside(Function):
 
     Explanation
     ===========
-
+Heac
     Heaviside function has the following properties:
 
     1) $\frac{d}{d x} \theta(x) = \delta(x)$
-    2) $\theta(x) = \begin{cases} 0 & \text{for}\: x < 0 \\ \text{undefined} &
+    2) $\theta(x) = \begin{cases} 0 & \text{for}\: x < 0 \\ \frac{1}{2} &
        \text{for}\: x = 0 \\1 & \text{for}\: x > 0 \end{cases}$
     3) $\frac{d}{d x} \max(x, 0) = \theta(x)$
 
     Heaviside(x) is printed as $\theta(x)$ with the SymPy LaTeX printer.
 
-    Regarding to the value at 0, Mathematica defines $\theta(0)=1$, but Maple
-    uses $\theta(0) = \text{undefined}$. Different application areas may have
-    specific conventions. For example, in control theory, it is common practice
-    to assume $\theta(0) = 0$ to match the Laplace transform of a DiracDelta
-    distribution.
+    The value at 0 is set differently in different fields. SymPy now uses 1/2,
+    which is a convention from electronics and signal processing. (Up to
+    SymPy 1.8, the default was that the value at 0 is undefined.)
```

Comment:
It would be good to use the versionchanged sphinx directive:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged

Reply:
OK!